client/src/components/header.jsx

```diff
@@ -0,0 +1,84 @@
+import styled, { css } from 'styled-components';
+import React, {Component} from 'react'
+import DollarSignsAndCategories from './dollarsignsandCategories.jsx';
+import Categories from './categories.jsx';
+import StarsAndReviews from './stars.jsx'
+
+const Title = styled.h1`
+  font-size: 30px;
+  color: #333;
+  margin: 0 6px;
+`;
+
+const BizWrapper = styled.div`
+  width: 960px;
+  margin: 0 auto;
+  padding: 0 15px;
+  font-family: Helvetica Neue,Helvetica,Arial,sans-serif; 
+  font-weight: 700:
+`
+const ClaimedText = styled.div`
+  font-size: 14px;
+  display: inline-block;  
+`
+
+
+
+
+class Header extends React.Component {
+  constructor(props){
+    super (props)
+
+    this.state = {
+      businessData: {}
+    }
+  }
+
+  componentDidMount(){
+    // function that gets all search params
+    let parseQueryString = () => {
+      let str = window.location.search;
```

Comment:
using a global variable?

Reply:
window.location gives me the url of the current window the browser is opened in.
window.location.search gives me the search query which can then be used to make a component did mount ajax call. 

i'm not sure if this is the correct way to make the ajax request that dynamically renders content, but it looks like thats whats going on the actual website. If you have a better way to do it (or if you would like to discuss further) please let me know lol